File: atcoder/abc326a_2up3down.hs
-- vicfred
-- https://atcoder.jp/contests/abc326/tasks/abc326_a
-- implementation
main :: IO ()
main = do
  [x, y] <- map read . words <$> getLine :: IO [Int]
  if (y > x && y - x <= 2) || (x > y && x - y <= 3) then
    putStrLn "Yes"
  else
    putStrLn "No"

File: atcoder/abc326A_2up3down.cpp
// vicfred
// https://atcoder.jp/contests/abc326/tasks/abc326_a
// implementation
#include <iostream>

using namespace std;

int main() {
  int X, Y;
  cin >> X >> Y;
  if(X - Y < -2 or X - Y > 3) {
    cout << "No" << endl;
  } else {
    cout << "Yes" << endl;
  }
  return 0;
}
